datastore/db/bytebase/FertiScan/schema_0.0.6.sql

```diff
@@ -0,0 +1,238 @@
+
+    CREATE TABLE "fertiscan_0.0.6"."users" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "email" text NOT NULL UNIQUE,
+    "registration_date" timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" timestamp
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."picture_set" (
+    "id" uuid NOT NULL DEFAULT uuid_generate_v4() PRIMARY KEY,
+    "picture_set" json NOT NULL,
+    "owner_id" uuid NOT NULL REFERENCES "fertiscan_0.0.6".users(id),
+    "upload_date" date NOT NULL DEFAULT current_timestamp,
+    "name" text
+    );
+        
+    CREATE TABLE "fertiscan_0.0.6"."picture" (
+    "id" uuid NOT NULL DEFAULT uuid_generate_v4() PRIMARY KEY,
+    "picture" json NOT NULL,
+    "picture_set_id" uuid NOT NULL REFERENCES "fertiscan_0.0.6".picture_set(id),
+    "verified" boolean NOT NULL DEFAULT false,
+    "upload_date" TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."province" (
+    "id" SERIAL PRIMARY KEY,
+    "name" text UNIQUE NOT NULL
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."region" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "province_id" int REFERENCES "fertiscan_0.0.6".province(id),
+    "name" text NOT NULL
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."location" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "name" text,
+    region_id uuid References "fertiscan_0.0.6".region(id),
+    "address" text NOT NULL
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."responsable" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "name" text NOT NULL,
+    "website" text,
+    "phone_number" text,
+    "location_id" uuid REFERENCES "fertiscan_0.0.6".location(id)
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."sample" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "number" uuid,
+    "collection_date" date,
+    "location" uuid REFERENCES "fertiscan_0.0.6".location(id)
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."specification" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "humidity" float,
+    "ph" float,
+    "solubility" float,
+    "edited" boolean,
+    "label_id" uuid REFERENCES "fertiscan_0.0.6".label_information(id)
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."unit" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "unit" text NOT NULL,
+    "to_si_unit" float
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."metric" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "value" float NOT NULL,
+    "unit_id" uuid REFERENCES "fertiscan_0.0.6".unit(id),
+    "edited" boolean
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."element_compound" (
+    "id" SERIAL PRIMARY KEY,
+    "number" int NOT NULL,
+    "name_fr" text NOT NULL,
+    "name_en" text NOT NULL,
+    "symbol" text NOT NULL UNIQUE
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."label_information" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "lot_number" text,
+    "npk" text,
```

Comment:
```suggestion

```


If there's already the N, P, K individually I don't think it's needed.

Reply:
npk as a text could be 10N20P30K or 10-20-30. Saving it as text helps us to really showcase whats printed on the label while keeping the numeric value for further filtering